src/modules/AccordionPanelGrommet.js

```diff
@@ -0,0 +1,143 @@
+import classnames from 'classnames';
+import Anchor from 'grommet/components/Anchor';
+import Box from 'grommet/components/Box';
+import Heading from 'grommet/components/Heading';
+import ListItem from 'grommet/components/ListItem';
+import Paragraph from 'grommet/components/Paragraph';
+import Tile from 'grommet/components/Tile';
+import Tiles from 'grommet/components/Tiles';
+import OpenIcon from 'grommet/components/icons/base/Add';
+import CloseIcon from 'grommet/components/icons/base/Subtract';
+
+const CLASS_ROOT = 'accordion-panel-grommet';
+
+export default class AccordionPanelGrommet extends Component {
+  constructor (props) {
+    super(props);
+
+    this._renderCTA = this._renderCTA.bind(this);
+    this._renderResources = this._renderResources.bind(this);
+    this._onClickPanel = this._onClickPanel.bind(this);
+
+    this.state = {
+      isOpen: false
+    };
+  }
+
+  _renderCTA () {
+    let { ctaThumbnail } = this.props;
+    // let { ctaThumbnail, ctaVideo } = this.props;
+    let { headline, subHeadline, link, linkText } = this.props.ctaText;
+    // TODO: add video overlay functionality for ctaVideo
+
+    return (
+      <Box direction="row" pad={{vertical: "medium"}}>
+        <img src={ctaThumbnail} className="accordion-cta-image" />
+        <Box pad="medium">
+          <Heading tag="h3" margin="none">{headline}</Heading>
+          <Paragraph margin="none">{subHeadline}</Paragraph>
+          <Anchor href={link} primary={true} label={linkText} />
+        </Box>
+      </Box>
+    );
+  }
+
+  _renderResources () {
+    let { resources } = this.props;
+    let resourceTiles;
+
+    if (resources) {
+      resourceTiles = resources.map(({ thumbnail, headline, subHeadline, link, linkText }, i) => {
+        return (
+          <Tile key={i} direction="row">
+            <img src={thumbnail} className="resource-image" />
+            <Box pad="medium">
+              <Heading tag="h6" margin="none">{headline}</Heading>
+              <Paragraph margin="none">{subHeadline}</Paragraph>
+              <Anchor href={link} primary={true} label={linkText} />
+            </Box>
+          </Tile>
+        );
+      });
+
+      return (
+        <Box pad={{vertical: "medium"}}>
+          <Heading tag="h3">Resources</Heading>
+          <Tiles fill={true}>
+            {resourceTiles}
+          </Tiles>
+        </Box>
+      );
+    }
+
+    return null;
+  }
+
+  _onClickPanel () {
+    let { isOpen } = this.state;
+    this.setState({ isOpen : !isOpen });
+  }
+
+  _renderPanelContent () {
+    let { headline, subHeadline } = this.props;
+    let { isOpen } = this.state;
+    if (isOpen) {
+      return (
+        <Box full="horizontal" pad={{vertical: "medium"}}>
+          <Heading tag="h3" margin="none" >{headline}</Heading>
+          <Paragraph margin="none">{subHeadline}</Paragraph>
+          {this._renderCTA()}
+          {this._renderResources()}
+        </Box>
+      );
+    }
+
+    return null;
```

Comment:
Just a note, Grommet core team likes to use `undefined` instead of `null`. As for this line, I think we can remove this and it will be okay.